src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/rest/LSFRestJobManager.groovy

```diff
@@ -494,54 +472,50 @@ class LSFRestJobManager extends BatchEuphoriaJobManagerAdapter {
         jobInfo.setTimeUnknownState(timeSummary.getProperty("unknownTime") ? Duration.ofSeconds(Math.round(Double.parseDouble(timeSummary.getProperty("unknownTime").toString())), 0) : null)
         jobInfo.setTimeSystemSuspState(timeSummary.getProperty("ssuspTime") ? Duration.ofSeconds(Math.round(Double.parseDouble(timeSummary.getProperty("ssuspTime").toString())), 0) : null)
         jobInfo.setRunTime(timeSummary.getProperty("runTime") ? Duration.ofSeconds(Math.round(Double.parseDouble(timeSummary.getProperty("runTime").toString())), 0) : null)
-        job.setJobInfo(jobInfo)
+
     }
 
     @Override
-    void addJobStatusChangeListener(BEJob job) {
-        synchronized (jobStatusListeners) {
-            jobStatusListeners.put(job.getJobID().getId(), job)
-        }
+    Map<BEJobID, GenericJobInfo> queryExtendedJobStateById(List<BEJobID> jobIds) {
+        Map<BEJobID, GenericJobInfo> jobDetailsResult = getJobDetails(jobIds)
+        updateJobStatistics(jobDetailsResult)
+        return jobDetailsResult
     }
 
     @Override
-    Map<String, JobState> queryJobStatusById(List<String> jobIds, boolean forceUpdate = false) {
-        return getJobStats(jobIds)
+    String getLogFileWildcard(BEJob job) {
+        return null
     }
 
     @Override
-    Map<String, JobState> queryJobStatusAll(boolean forceUpdate = false) {
-        return getJobStats(null)
+    String getStringForQueuedJob() {
+        return null
     }
 
     @Override
-    Map<BEJob, GenericJobInfo> queryExtendedJobState(List<BEJob> jobs, boolean forceUpdate) {
+    String getStringForJobOnHold() {
+        return null
+    }
 
-        Map<String, GenericJobInfo> queriedExtendedStates = queryExtendedJobStateById(jobs.collect {
-            it.getJobID().toString()
-        }, false)
-        return (Map<BEJob, GenericJobInfo>) queriedExtendedStates.collectEntries { Map.Entry<String, GenericJobInfo> it -> [jobs.find { BEJob temp -> temp.getJobID().getId() == it.key }, (GenericJobInfo) it.value] }
+    @Override
+    String getStringForRunningJob() {
+        return null
     }
 
+    @Override
+    String getQueueVariable() {
+        return null
```

Comment:
Should that not be the same as for the LSFJobManager? It ist a variable that may be used in the job itself and the JobManager has this as method to allow client code to use the right variable dependent on the manager type.

Reply:
fixed